packages/maskbook/src/plugins/Voicechat/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const PLUGIN_IDENTIFIER = 'com.maskbook.voicechat'
+export const VOICECHAT_GUN_SERVER = 'com.maskbook.plugin.voicechat'
+export const VOICECHAT_META_KEY_1 = 'com.maskbook.voicechat:1'
+
+export const SIGNALING_SERVER = 'http://localhost:3000'
+export const ICE_SERVERS = [{ urls: 'stun:stun.l.google.com:19302' }]
```

Comment:
cc @yisiliu is this stun server ok?

Reply:
actually I think we can use it for demo but in production, I'd prefer a self-hosted stun.

Reply:
https://github.com/coturn/coturn

Reply:
You could safely use the google provided one unless you expect a TON of users hitting the voicechat plugin

Reply:
@yisiliu when deploy my stun server?